pkgs/build-support/make-darwin-bundle/write-darwin-bundle.nix

```
@@ -17,23 +17,30 @@ in writeScriptBin "write-darwin-bundle" ''
 
     readonly prefix=$1
     readonly name=$2
-    readonly exec=$3
+    readonly execName=''${3%% *} # Before the first space
+    [[ $3 =~ " " ]] && readonly execArgs=''${3#* } # Everything after the first space
     readonly icon=$4.icns
     readonly squircle=''${5:-1}
     readonly plist=$prefix/Applications/$name.app/Contents/Info.plist
+    readonly binary="$prefix/bin/$execName"
+    readonly bundleExecutable="$prefix/Applications/$name.app/Contents/MacOS/$name"
```

Quoting isn’t needed for variable definition.

```suggestion
    readonly binary=$prefix/bin/$execName
    readonly bundleExecutable=$prefix/Applications/$name.app/Contents/MacOS/$name
```

---

Fixed :+1: